### ISSUE TITLE
Search: Sort Order Lucene Search Adapter is wrong

### ISSUE THREAD

**Issue by ampaze** [sulu/sulu#6521]
| Q | A
| --- | ---
| Sulu Version | 2.3.x
| PHP Version | 8.0.15

I hope I am missing something, but according to https://github.com/massiveart/MassiveSearchBundle/pull/31 the Lucene search results are resorted (because Lucene apparently does not support this) - unfortunately it is sorted lowest to highest score...?

# Actual Behavior

The Lucene search results are ordered lowest to highest

# Expected Behavior

The Lucene search results are ordered highest to lowest

# Steps to Reproduce

Make a search and dump the search result and look at the individual scores of the hits.
```php
        foreach ($hits as $hit) {
            dump($hit->getScore());
        }
```

# Possible Solutions

Fix the sort order somewhere, maybe directly in the PR mentioned above.

**Comment by alexander-schranz** [comment]
The function looks correct for me it output the highest value first:

See as example: https://3v4l.org/Hk3ul#v7.4.28

`$hits = [2, 1, 4];` -> `[4, 2, 1]`

**Comment by alexander-schranz** [comment]
Looks like there was an accidently change to spaceship operator which did crash this:
https://github.com/massiveart/MassiveSearchBundle/blob/167ba9ae8aef57a7100a3890705b6d3af7f511a2/Search/Adapter/ZendLuceneAdapter.php#L273

**Comment by ampaze** [comment]
Yes, the spaceship operator one is the one that gets installed, at least for me.

Corrected Sample https://3v4l.org/Vp01n#v7.4.28

**Comment by alexander-schranz** [comment]
Will be fixed in the next massive search bundle release: https://github.com/massiveart/MassiveSearchBundle/pull/166

Thx for report!

**Comment by alexander-schranz** [comment]
Released in massive search bundle 2.6.5: https://github.com/massiveart/MassiveSearchBundle/commit/5b06f81092e0b982e1a63e93a93de8b29fe44a13

**Comment by ampaze** [comment]
Thanks!
